### ISSUE TITLE
HTML elements should not have redundant roles

### ISSUE THREAD

**Issue by gnpivo** [w3c/wai-tutorials#780]
Hi. The [Page Regions in HTML5 Using WAI-ARIA](https://www.w3.org/WAI/tutorials/page-structure/regions/#accessupport) section suggests using both the HTML element and its corresponding role:

```html
<header role="banner">…</header>
<main role="main">…</main>
<nav role="navigation">…</nav>
<footer role="contentinfo">…</footer>
```

However, the [ARIA in HTML](https://www.w3.org/TR/html-aria/#avoid-specifying-redundant-roles) recommendation specifically discourages this usage:

> The following example illustrates a [button](https://html.spec.whatwg.org/multipage/form-elements.html#the-button-element) element which has also been provided an explicit role=button. Specifying this role is unnecessary, as a "button" element is already exposed with an implicit button role. In practice this particular instance of redundancy will likely not have unforeseen side effects, other than unnecessarily making the markup more verbose, and incorrectly signaling to other authors that this practice is useful. Please review the section [3.3 Be cautious of side effects](https://www.w3.org/TR/html-aria/#side-effects) for an example of where specifying unnecessary roles can be problematic.
>
> ```html
> 
> <button role="button">...</button>
> ```

I believe the tutorial should be modified to conform with the recommendation by not encouraging the use of redundant roles.

**Comment by brianelton** [comment]
I agree. This section could probably be removed completely. The [HTML4 section](https://www.w3.org/WAI/tutorials/page-structure/regions/#accessupport-html4) provides sufficient information for the use of ARIA roles when HTML5 can not be used.
